src/Data/Product/Relation/Binary/Lex/Strict.agda

```diff
@@ -200,6 +199,30 @@ module _ {_<₁_ : Rel A ℓ₁} {_<₂_ : Rel B ℓ₂} where
     ×-acc acc₁ (acc rec₂) (u , v) (inj₂ (refl , v<y))
       = acc (×-acc acc₁ (rec₂ v v<y))
 
+module _ {_≈₁_ : Rel A ℓ₁} {_<₁_ : Rel A ℓ₂} {_<₂_ : Rel B ℓ₃} where
+
+  private
+    _<ₗₑₓ_ = ×-Lex _≈₁_ _<₁_ _<₂_
+
+  ×-wellFounded' : IsPartialEquivalence _≈₁_ →
```

Comment:
If we swap the order of these proofs can we then get the propositional proof for free from the setoid proof?

Reply:
```agda
×-wellFounded : WellFounded _<₁_ →
                  WellFounded _<₂_ →
                  WellFounded _<ₗₑₓ_
×-wellFounded = ×-wellFounded' ≡.sym ≡.trans (≡.respʳ _<₁_)
```

Is this what you had in mind ?

Reply:
Yup! Thanks.